### PR TITLE
Adds initialization options for HTML IAMs and SDK Flavor

### PR DESCRIPTION
**Pull request by briancaw**

### CODE REVIEW

File: AppboyKit.js [hunk 1]
```diff
@@ -223,6 +223,8 @@
                 isTesting = testMode;
                 // 30 min is Appboy default
                 options.sessionTimeoutInSeconds = forwarderSettings.ABKSessionTimeoutKey || 1800;
+                options.sdkFlavor = “mparticle”;
```
Comment (samdozor) [inline AppboyKit.js]: these seem like curly/smart quotes - can you please update to regular quotes?

File: AppboyKit.js [hunk 2]
```diff
@@ -223,6 +223,8 @@
...
+                options.enableHtmlInAppMessages = true;
```
Comment (samdozor) [inline AppboyKit.js]: should we have a setting for this? What's the impact if a customer doesn't want this?

Comment (samdozor) [inline AppboyKit.js]: Let's add a server setting for this, called `enableHtmlInAppMessages`.

File: AppboyKit.js (hunk 1 above)
Comment (briancaw) [inline AppboyKit.js]: updated

File: AppboyKit.js (hunk 2 above)
Comment (briancaw) [inline AppboyKit.js]: updated

File: AppboyKit.js [hunk 3]
```diff
@@ -223,6 +223,8 @@
                 isTesting = testMode;
                 // 30 min is Appboy default
                 options.sessionTimeoutInSeconds = forwarderSettings.ABKSessionTimeoutKey || 1800;
+                options.sdkFlavor = 'mparticle';
+                options.enableHtmlInAppMessages = forwarderSettings.enableHtmlInAppMessages;
```
Comment (samdozor) [inline AppboyKit.js]: @briancaw actually can you update this to:
```
options.enableHtmlInAppMessages = forwarderSettings.enableHtmlInAppMessages == 'True'
```

**Pull request merged**